stablehlo/compatibility/transforms/MapStablehloToVersionedStablehlo.h

```diff
@@ -0,0 +1,176 @@
+/* Copyright 2022 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef MLIR_HLO_DIALECT_MHLO_TRANSFORMS_MAP_STABLEHLO_TO_HLO_OP_H
+#define MLIR_HLO_DIALECT_MHLO_TRANSFORMS_MAP_STABLEHLO_TO_HLO_OP_H
```

Comment:
Fixed. Also found a few typos when trying to scan for other headers guards that may be misnamed (#668).

Reply:
Also had a thought regarding CI for this - I was having issues running clang-tidy on github CI because it tends to be flaky, however it may prove beneficial to run a more limited tidy CI. Maybe if we only checked header guards, and gradually built up from there depending on flakiness, we could get some useful CI in the near future.